src/Lykke.Service.BlockchainSettings.Services/BlockchainValidationService.cs

```diff
@@ -42,15 +45,15 @@ public async Task<bool> ValidateServiceUrlAsync(string serviceUrl)
 
             try
             {
-                using (var client = new Lykke.Service.BlockchainApi.Client.BlockchainApiClient(_log, serviceUrl, 3))
+                using (var client = new BlockchainApiClient(_logFactory, serviceUrl, 3))
                 {
                     var isAlive = await client.GetIsAliveAsync();
                     isUrlValid = isAlive != null;
                 }
             }
             catch (Exception e)
             {
-                _log.WriteInfo(nameof(ValidateHotwalletAsync), $"{serviceUrl}", "Could not check api validity");
+                _log.Info(nameof(ValidateHotwalletAsync), "Could not check api validity", $"{serviceUrl}");
```

Comment:
This was before your modification, but exeption just swalloed here - it's bad practice. Exception should be either logged or rethrowed